Sources/MultipartKit/FormDataDecoder.swift

```diff
@@ -229,6 +229,11 @@ private extension MultipartFormData {
         else {
             return try T(from: _FormDataDecoder(codingPath: codingPath, data: self))
         }
-        return Convertible.init(multipart: part) as! T
+        guard
+            let converted = Convertible.init(multipart: part) as! T?
```

Comment:
Should this not be
```suggestion
            let converted = Convertible.init(multipart: part) as? T?
```
To avoid crashing if the decoding fails?

Reply:
@0xTim the crash was already avoided but you're right, it can be written that way as well. The reason force unwrap works here is that `Convertible.init(multipart:)` returns a `Convertible?` which we know to be `T?`. The previous implementation was doing `as! T` which only worked if the the initializer did not return `nil`.
I now removed the force unwrap and deduplicated this conversion since more or less the same thing was done twice.